How should global parameters be passed

I am trying to pass global parameters to stories. The non-nuxtjs way is to add the desired data in .storybook/preview.js, but in this case the respective file ./nuxt-storybook/storybook/preview.js is overwritten in build time. I have also tried to pass them as a storybook option in nuxt.config.js without success.

Unfortunately there is no reference in the documentation. How should we handle this?

Sidenote: Addon options seem not to work as documented as well.

The documentation's example:
`export default {
  storybook: {
    addons: [
      {
        name: '@storybook/preset-scss',
        options: {
          cssLoaderOptions: {
              modules: true,
              localIdentName: '[name]__[local]--[hash:base64:5]',
          }
        }
      }
    ]
  }
}`

It throws the error: Cannot find module '[object Object]' the moment you insert an object instead of a string in the addons[] array.

--- a/docs/content/en/options.md
+++ b/docs/content/en/options.md
@@ -109,3 +109,24 @@ export default {
   }
 }
 ```
+
+## `parameters`
+> Parameters are a set of static, named metadata about a story, typically used to control the behavior of Storybook features and addons.[*](https://storybook.js.org/docs/react/writing-stories/parameters)
+
+Customize Storybook's global parameters.  
+
+```js{}[nuxt.config.js]
+export default {
+  storybook: {
+    parameters: {
+      backgrounds: {
+        default: 'white',
+        values: [
+          { name: 'white', value: '#ffffff' },
+          { name: 'gray', value: '#aaaaaa' },
+        ],
+      },
+    }
+  }
+}
+```

--- a/package.json
+++ b/package.json
@@ -47,6 +47,7 @@
     "@nuxtjs/proxy": "^2.0.1",
     "@nuxtjs/tailwindcss": "^3.0.2",
     "@storybook/addon-docs": "^6.0.20",
+    "@storybook/addon-essentials": "^6.0.20",
     "@storybook/addon-knobs": "^6.0.20",
     "@types/ci-info": "latest",
     "@types/inquirer": "latest",

--- a/playground/nuxt.config.js
+++ b/playground/nuxt.config.js
@@ -34,7 +34,7 @@ export default {
     },
     parameters: {
       backgrounds: {
-        default: 'twitter',
+        default: 'white',
         values: [
           { name: 'white', value: '#ffffff' },
           { name: 'gray', value: '#aaaaaa' },

--- a/playground/nuxt.config.js
+++ b/playground/nuxt.config.js
@@ -18,11 +18,28 @@ export default {
     moduleName: "@nuxtjs/storybook",
   },
   storybook: {
+    addons: [
+      {
+        name: '@storybook/addon-essentials',
+        options: {
+          actions: false,
+        }
+      }
+    ],
     stories: [
       '~/stories/**/*.stories.@(ts|js)'
     ],
     webpackFinal (config) {
       return config
+    },
+    parameters: {
+      backgrounds: {
+        default: 'twitter',
+        values: [
+          { name: 'white', value: '#ffffff' },
+          { name: 'gray', value: '#aaaaaa' },
+        ],
+      },
     }
   }
 }

--- a/src/index.ts
+++ b/src/index.ts
@@ -157,7 +157,8 @@ export async function eject (options: StorybookOptions) {
 function nuxtStorybookOptions (options) {
   const nuxtStorybookConfig = Object.assign({
     stories: [],
-    addons: []
+    addons: [],
+    parameters: {}
   }, options.storybook)
 
   nuxtStorybookConfig.configDir = path.resolve(options.rootDir, 'storybook')

--- a/storybook/preview.js
+++ b/storybook/preview.js
@@ -3,3 +3,5 @@ import '~storybook';
 
 <% if (options.nuxtOptions.components) { %>import * as components from '../components';
 Object.keys(components).forEach(name => Vue.component(name, components[name]))<% } %>
+
+export const parameters = <%= devalue(options.parameters || {}) %>

--- a/storybook/preview.js
+++ b/storybook/preview.js
@@ -4,4 +4,4 @@ import '~storybook';
 <% if (options.nuxtOptions.components) { %>import * as components from '../components';
 Object.keys(components).forEach(name => Vue.component(name, components[name]))<% } %>
 
-export const parameters = <%= devalue(options.parameters || {}) %>
+export const parameters = <%= devalue(options.parameters) %>

